router/build.gradle

```diff
@@ -30,9 +30,6 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$versions.kotlin"
   implementation "io.reactivex.rxjava2:rxjava:$versions.rxJava"
   implementation "com.jakewharton.timber:timber:$versions.timber"
-  testImplementation "pl.pragmatists:JUnitParams:$versions.junitParams"
+  testImplementation "com.github.vinaysshenoy:JUnitParams:$versions.junitParams"
```

Comment:
Why was forking needed?

Reply:
The reasoning is in the commit message.

Copying it over here:
```
The main dependency has a bug where annotations added on test methods
are not forwarded to any test rules (Pragmatists/JUnitParams#151),
which breaks the `ExpectedUnsubscribed` annotation used for hinting to
the `RxErrorsRule` that we expect a certain number of `Completable`
instances to not be subscribed to.

There is a PR to fix it (Pragmatists/JUnitParams#158),
but this has been inactive for over a year, so I forked the repository
and applied the fix myself and published it via Jitpack.
```